cub/csv/csv_reader.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 ZTE corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "cub/csv/csv_reader.h"
+
+#include "cub/log/log.h"
+#include "cub/string/str_utils.h"
+
+namespace cub {
+
+CSVReader::CSVReader(const std::string& filename, const std::string& delm)
+    : file_name(filename), delimeter(delm), stream(file_name) {
+}
+
+bool CSVReader::read() {
+  if (!stream.is_open()) {
+    ERR_LOG << "Input file '" << file_name << "' can't open!";
+    return false;
+  }
+
+  std::string line = "";
+  if (std::getline(stream, line)) {
+    cols = cub::strutils::split(line, delimeter);
+    for (auto& s : cols) {
+      s.erase(s.find_last_not_of(" \n\r\t") + 1);
+    }
+  }
```

Comment:
Handle quotes, commas and newlines properly according to <https://tools.ietf.org/html/rfc4180>.

Reply:
I will make an issue to fix it